src/faction_camp.cpp

```diff
@@ -191,8 +191,8 @@ struct mass_volume {
 
 namespace base_camps
 {
-static const std::string recover_ally_string = "Recover Ally, ";
-static const std::string expansion_string = " Expansion";
+static const std::string recover_ally_string = _( "Recover Ally, " );
```

Comment:
You only need to use `translate_marker` so the strings are correctly extracted. The strings are already translated where they are used.

Reply:
As Qrox noted, the string is translated in both locations where it's used. The second string is now used only in one of those locations, and so can be turned into a literal at that place (around line 787).

Reply:
Hey, thanks for the tip, I see GitHub action says something about it too; what are the differences between the functions? Is it documented somewhere? I'll try to take more deepr look when I get home!

Reply:
`doc/TRANSLATING.md` has some explanation of the translation functions. I also added some explanation of the clang-tidy check in #56509.

Reply:
> You only need to use `translate_marker` so the strings are correctly extracted. The strings are already translated where they are used. Or better yet, use `translation` so it won't be unmarked nor accidentally translated twice like this.

I finally grasped my head around it following the expanded explanation by @BrettDong on Discord. I went with the second route, per your suggestion – hopefully I did it all right!

Reply:
You should also replace the `_( str )` calls with `str.translated()`.